User Story Standards

```diff
@@ -0,0 +1,238 @@
+<span style="color:#191970">**EIS-M Project User Story Standards** 
+---
+The goal of this document is to establish user story minimum acceptable standards that define a common set of core documentation standards that can be used consistently across multiple product teams when developing features for an integrated eligibility system.
+
+Defining minimum acceptable documentation standards enables product teams to communicate and document epics, features and user stories in a manner that is consistent across teams but flexible and efficient, to support rapid development and deployments.
+
+Our consistency in approach will assist us with our primary objective to add value to the user in each Sprint. The prioritization of the user stories will be crucial as we discover user needs with each code delivery.
+
+<span style="color:#191970">**Product Teams**
+---
+Each team is a cross functional group of members comprised of business, policy, security, procurement and technical specialists tasked with implementing features to support goals and/or outcomes identified by the primary product team in accordance to its vision.  
+
+<span style="color:#191970">**DOH Vision Statement**
+---
+Alaska Department of Health (DOH) is developing a modern, integrated eligibility system that enables staff to more efficiently issue correct and timely benefits to Alaskans who need help meeting their basic needs.  Using agile development and modular procurements, DOH has the goal of implementing an eligibility system that not only meets state and federal standards, but is user friendly for our clients, our eligibility staff, and our technical staff.
+
+Implementing our project’s vision can be decomposed into 4 work levels for execution:  Goal/Outcome -> Epics -> Features -> User Stories
+
+<center>
```

Comment:
Remove

Reply:
I think i got rid of all the ADO md stuff....